src/Prompt.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+const Prompt = ({question, image, options}) => {
+    return (
+        <div>
+            {question}
+            {image}
+            {JSON.stringify(options)}
+        </div>
+    );
+}
+
+export default Prompt
```

Comment:
think the file structure is good for now.  I'm thinking we should a components directory under src and this should go in there. just to keep it a little tidier